### ISSUE TITLE
Unable to join paths to a single curve

### ISSUE THREAD

**Issue by makamto** [otfried/ipe#537]
<img width="949" alt="image" src="https://github.com/user-attachments/assets/92c3a8cb-8d65-4655-8b69-b01dc4ec85ae">

**Comment by otfried** [comment]
This should just work.

![image](https://github.com/user-attachments/assets/a8a8151a-6313-4fce-aa79-7651e6367eea)

**Comment by makamto** [comment]
I wanted to let you know that I was able to solve my issue. However, before, while using Spline to create paths, I received an error message that said, "Unable to join paths to a single curve." I think this might have happened because the y-axis line was included in "join path". To fix it, I separated my work into three layers: filled, axis, and function (curve), and that worked.

Thank you for your quick reply to my last question. Since I am new to Ipe, I apologize if I took up too much of your time. I would love to know how did you draw the diagram above in Ipe, as I want to learn more an efficient workflow.

<img width="995" alt="image" src="https://github.com/user-attachments/assets/b6925a8f-5941-4f81-a281-cc28705c87ea">
